denizen_scripts/HeroCraft/mobs/suffixes/teleporting.dsc

```diff
@@ -2,5 +2,8 @@ custom_mob_suffix_teleporting:
   Type: world
   debug: false
   events:
-    after entity_flagged:teleporting targets entity:
-        - teleport <context.entity> <context.target.location> if:<util.random_chance[25]>
\ No newline at end of file
+    after entity_flagged:teleporting targets entity chance:25:
+        - playsound <context.target.location> sound:entity_enderman_teleport volume:2.0
+        - playeffect <context.target.eye_location> effect:item_crack quantity:100 offset:0.5,0.5,0.5 special_data::ender_eye
+        - playeffect <context.entity.eye_location> effect:item_crack quantity:100 offset:0.5,0.5,0.5 special_data::ender_eye
```

Comment:
Why double?

Reply:
one at target location and one at origin